Bazel 5.1.1 with openjdk17 server

### Description of the bug:


If using openjdk17 as bazel server runtime some actions may fail
```
java.lang.ExceptionInInitializerError
	at com.google.devtools.build.lib.actions.ParameterFile.writeContent(ParameterFile.java:118)
	at com.google.devtools.build.lib.actions.ParameterFile.writeParameterFile(ParameterFile.java:111)
	at com.google.devtools.build.lib.analysis.actions.ParameterFileWriteAction$ParamFileWriter.writeOutputFile(ParameterFileWriteAction.java:170)
	at com.google.devtools.build.lib.exec.FileWriteStrategy.beginWriteOutputToFile(FileWriteStrategy.java:58)
	at com.google.devtools.build.lib.analysis.actions.FileWriteActionContext.beginWriteOutputToFile(FileWriteActionContext.java:49)
	at com.google.devtools.build.lib.analysis.actions.AbstractFileWriteAction.beginExecution(AbstractFileWriteAction.java:66)
	at com.google.devtools.build.lib.actions.Action.execute(Action.java:133)
	at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor$5.execute(SkyframeActionExecutor.java:907)
	at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor$ActionRunner.continueAction(SkyframeActionExecutor.java:1076)
	at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor$ActionRunner.run(SkyframeActionExecutor.java:1031)
	at com.google.devtools.build.lib.skyframe.ActionExecutionState.runStateMachine(ActionExecutionState.java:152)
	at com.google.devtools.build.lib.skyframe.ActionExecutionState.getResultOrDependOnFuture(ActionExecutionState.java:91)
	at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor.executeAction(SkyframeActionExecutor.java:492)
	at com.google.devtools.build.lib.skyframe.ActionExecutionFunction.checkCacheAndExecuteIfNeeded(ActionExecutionFunction.java:856)
	at com.google.devtools.build.lib.skyframe.ActionExecutionFunction.computeInternal(ActionExecutionFunction.java:349)
	at com.google.devtools.build.lib.skyframe.ActionExecutionFunction.compute(ActionExecutionFunction.java:169)
	at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:590)
	at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor$WrappedRunnable.run(AbstractQueueVisitor.java:382)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make java.lang.String(byte[],byte) accessible: module java.base does not "opens java.lang" to unnamed module @5e3c3cb
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
	at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
	at java.base/java.lang.reflect.Constructor.checkCanSetAccessible(Constructor.java:188)
	at java.base/java.lang.reflect.Constructor.setAccessible(Constructor.java:181)
	at com.google.devtools.build.lib.unsafe.StringUnsafe.<init>(StringUnsafe.java:75)
	at com.google.devtools.build.lib.unsafe.StringUnsafe.initInstance(StringUnsafe.java:56)
	at com.google.devtools.build.lib.unsafe.StringUnsafe.<clinit>(StringUnsafe.java:37)
	... 21 more
```

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.


Haven't minimized this yet, also not yet tested with newer bazel releases

### Which operating system are you running Bazel on?


NixOS

### What is the output of `bazel info release`?


release 5.1.1- (@non-git)

### If `bazel info release` returns `development version` or `(@non-git)`, tell us how you built Bazel.


NixOS nixpkgs

### What's the output of `git remote get-url origin; git rev-parse master; git rev-parse HEAD` ?


_No response_

### Have you found anything relevant by searching the web?

_No response_

### Any other information, logs, or outputs that you want to share?

https://openjdk.org/jeps/403
since java16 default is
> -illegal-access=deny disables all illegal-access operations except for those enabled by other command-line options, e.g., --add-opens.

Something like `--add-opens=java.base/java.lang=ALL-UNNAMED` for server jvm flags could be a workaround

somewhat related https://github.com/bazelbuild/bazel/issues/5599

reflection is still present in latest bazel source code https://github.com/bazelbuild/bazel/blob/master/src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java

I'm not dmivankov, but I tried compling v 5.2.0 with openjdk17 and am getting the same crash. Here's a full build log:

```sh

Building Bazel from scratch../opt/openjdk-bin-17.0.3_p7/bin/javac -classpath derived/jars/zstd-jni/libzstd-jni.jar:derived/jars/com_google_protobuf/java/core/liblite.jar:derived/jars/com_google_protobuf/java/core/libcore.jar:derived/jars/com_google_protobuf/java/util/libutil.jar:third_party/checker_framework_annotations/checker-qual-3.2.0-sources.jar:third_party/checker_framework_annotations/checker-qual-3.2.0.jar:third_party/javax_annotations/javax.annotation-api-1.3.2-sources.jar:third_party/javax_annotations/javax.annotation-api-1.3.2.jar:third_party/truth/truth-1.0.1.jar:third_party/bazel_bootstrap/libserialization-constant-annotation.jar:third_party/bazel_bootstrap/libserialization-constant-processor.jar:third_party/bazel_bootstrap/libserialization.jar:third_party/bazel_bootstrap/libautocodec-processor.jar:third_party/bazel_bootstrap/libunsafe-provider.jar:third_party/bazel_bootstrap/libautocodec-annotation.jar:third_party/bazel_bootstrap/libserialization-processor-util.jar:third_party/bazel_bootstrap/libregistered-singleton.jar:third_party/turbine/turbine_direct.jar:third_party/jcip_annotations/jcip-annotations-1.0-1.jar:third_party/apache_commons_pool2/commons-pool2-2.8.0.jar:third_party/apache_commons_compress/apache-commons-compress-1.19.jar:third_party/animal_sniffer/animal-sniffer-annotations-1.14.jar:third_party/flogger/flogger-system-backend-0.5.1.jar:third_party/flogger/flogger-0.5.1.jar:third_party/flogger/google-extensions-0.5.1.jar:third_party/jetbrains_annotations/annotations-13.0.jar:third_party/truth8/truth-java8-extension-1.0.1.jar:third_party/objenesis/objenesis-1_3.jar:third_party/api_client/google-api-client-1.22.0.jar:third_party/api_client/google-api-client-gson-1.22.0.jar:third_party/api_client/google-http-client-1.22.0-SNAPSHOT.jar:third_party/api_client/google-http-client-gson-1.22.0.jar:third_party/tomcat_annotations_api/tomcat-annotations-api-8.0.5.jar:third_party/jimfs/jimfs-1.1.jar:third_party/hamcrest/hamcrest-core-1.3.jar:third_party/diffutils/diffutils-1.3.0.jar:third_party/caffeine/caffeine-2.9.2.jar:third_party/checker_framework_javacutil/javacutil-3.2.0-sources.jar:third_party/checker_framework_javacutil/javacutil-3.2.0.jar:third_party/jsr330_inject/javax.inject.jar:third_party/java/jdk/langtools/jdk_compiler-src.jar:third_party/java/jdk/langtools/jdk_compiler.jar:third_party/java/jdk/langtools/java_compiler.jar:third_party/java/jdk/langtools/java_compiler-src.jar:third_party/java/jdk/langtools/javac-9+181-r4173-1.jar:third_party/java/proguard/proguard6.2.2/lib/proguard.jar:third_party/java/proguard/proguard6.2.2/lib/proguardgui.jar:third_party/java/proguard/proguard6.2.2/lib/retrace.jar:third_party/java/javapoet/javapoet-1.8.0.jar:third_party/java/android_databinding/v3_4_0/base.jar:third_party/java/android_databinding/v3_4_0/exec.jar:third_party/java/android_databinding/v3_4_0/x_common.jar:third_party/java/android_databinding/v3_4_0/compiler.jar:third_party/java/android_databinding/v3_4_0/compiler_common.jar:third_party/java/jacoco/org.jacoco.ant-0.8.7-sources.jar:third_party/java/jacoco/org.jacoco.agent-0.8.7.jar:third_party/java/jacoco/org.jacoco.ant-0.8.7-nodeps.jar:third_party/java/jacoco/org.jacoco.agent-0.8.7-sources.jar:third_party/java/jacoco/org.jacoco.core-0.8.7.jar:third_party/java/jacoco/jacocoagent-0.8.7.jar:third_party/java/jacoco/org.jacoco.report-0.8.7-sources.jar:third_party/java/jacoco/org.jacoco.ant-0.8.7.jar:third_party/java/jacoco/org.jacoco.report-0.8.7.jar:third_party/java/jacoco/org.jacoco.core-0.8.7-sources.jar:third_party/java/j2objc-annotations/j2objc-annotations-1.1.jar:third_party/java/jcommander/jcommander-1.48.jar:third_party/juniversalchardet/juniversalchardet-1.0.3.jar:third_party/guava/failureaccess-1.0.1.jar:third_party/checker_framework_dataflow/dataflow-errorprone-3.15.0.jar:third_party/checker_framework_dataflow/dataflow-shaded-3.11.0.jar:third_party/checker_framework_dataflow/dataflow-errorprone-3.15.0-sources.jar:third_party/checker_framework_dataflow/dataflow-shaded-3.11.0-sources.jar:third_party/jsr305/jsr-305.jar:third_party/asm/asm-util-9.2.jar:third_party/asm/asm-util-9.2-sources.jar:third_party/asm/asm-commons-9.2-sources.jar:third_party/asm/asm-tree-9.2.jar:third_party/asm/asm-analysis-9.2-sources.jar:third_party/asm/asm-commons-9.2.jar:third_party/asm/asm-analysis-9.2.jar:third_party/asm/asm-9.2-sources.jar:third_party/asm/asm-tree-9.2-sources.jar:third_party/asm/asm-9.2.jar:third_party/apache_velocity/velocity-1.7.jar:third_party/mockito/mockito-core-3.12.4.jar:third_party/allocation_instrumenter/java-allocation-instrumenter-3.3.0.jar:third_party/antlr/antlr4-4.5.3.jar:third_party/perfmark/perfmark-api-0.19.0.jar:third_party/error_prone/error_prone_core-2.9.0.jar:third_party/error_prone/threeten-extra-1.5.0.jar:third_party/error_prone/error_prone_type_annotations-2.9.0.jar:third_party/error_prone/error_prone_annotation-2.9.0.jar:third_party/error_prone/error_prone_annotations-2.9.0.jar:third_party/error_prone/error_prone_check_api-2.9.0.jar:third_party/ijar/test/libwrongcentraldir.jar:third_party/ijar/test/jar-without-manifest.jar:third_party/ijar/test/sealed/sealed.jar:third_party/ijar/test/nestmates/nestmates.jar:third_party/ijar/test/records/records.jar:third_party/ijar/test/jar-with-manifest.jar:third_party/ijar/test/jar-with-manifest-and-target-label.jar:third_party/javax_activation/javax.activation-api-1.2.0-sources.jar:third_party/javax_activation/javax.activation-api-1.2.0.jar:third_party/apache_commons_collections/commons-collections-3.2.2.jar:third_party/xz/xz-1.9.jar:third_party/apache_commons_lang/commons-lang-2.6.jar:third_party/junit/junit-4.13.jar:third_party/java-diff-utils/java-diff-utils-4.0.jar:third_party/rxjava3/rxjava-3.1.2.jar:third_party/apache_commons_logging/commons-logging-1.1.1.jar:third_party/jaxb/2.3.0/jaxb-core-2.3.0-sources.jar:third_party/jaxb/2.3.0/jaxb-core-2.3.0.jar:third_party/jaxb/2.3.0/jaxb-api-2.3.0-sources.jar:third_party/jaxb/2.3.0/jaxb-impl-2.3.0.jar:third_party/jaxb/2.3.0/jaxb-api-2.3.0.jar:third_party/jaxb/2.3.0/jaxb-impl-2.3.0-sources.jar:third_party/jaxb/jaxb-api-2.3.1.jar:third_party/jaxb/jaxb-api-2.3.1-patched.jar:third_party/jaxb/jaxb-api-2.3.1-sources.jar:third_party/hungarian_algorithm/software-and-algorithms-1.0.jar:third_party/hungarian_algorithm/software-and-algorithms-1.0-src.jar:third_party/kotlin_stdlib/kotlin-stdlib-jdk8-1.3.21.jar:third_party/kotlin_stdlib/kotlin-stdlib-1.3.21.jar:third_party/kotlin_stdlib/kotlin-stdlib-jdk7-1.3.21.jar:third_party/kotlin_stdlib/kotlin-stdlib-common-1.3.21.jar:third_party/apache_commons_io/commons-io-2.4.jar:third_party/reactive_streams/reactive-streams-1.0.3.jar:third_party/netty/netty-transport-native-unix-common-4.1.69.Final-linux-aarch_64.jar:third_party/netty/netty-handler-proxy-4.1.69.Final.jar:third_party/netty/netty-codec-http2-4.1.69.Final.jar:third_party/netty/netty-buffer-4.1.69.Final.jar:third_party/netty/netty-handler-4.1.69.Final.jar:third_party/netty/netty-transport-native-unix-common-4.1.69.Final-osx-x86_64.jar:third_party/netty/netty-transport-native-epoll-4.1.69.Final-linux-x86_64.jar:third_party/netty/netty-transport-native-unix-common-4.1.69.Final-osx-aarch_64.jar:third_party/netty/netty-transport-native-unix-common-4.1.69.Final.jar:third_party/netty/netty-transport-sctp-4.1.69.Final.jar:third_party/netty/netty-transport-native-kqueue-4.1.69.Final.jar:third_party/netty/netty-transport-native-unix-common-4.1.69.Final-linux-x86_64.jar:third_party/netty/netty-common-4.1.69.Final.jar:third_party/netty/netty-resolver-4.1.69.Final.jar:third_party/netty/netty-transport-native-epoll-4.1.69.Final.jar:third_party/netty/netty-resolver-dns-4.1.69.Final.jar:third_party/netty/netty-transport-4.1.69.Final.jar:third_party/netty/netty-codec-http-4.1.69.Final.jar:third_party/netty/netty-transport-native-epoll-4.1.69.Final-linux-aarch_64.jar:third_party/netty/netty-transport-native-kqueue-4.1.69.Final-osx-aarch_64.jar:third_party/netty/netty-transport-native-kqueue-4.1.69.Final-osx-x86_64.jar:third_party/netty/netty-codec-4.1.69.Final.jar:third_party/auto/auto-service-annotations-1.0-rc7.jar:third_party/auto/auto-value-1.8.2.jar:third_party/auto/auto-service-1.0-rc7.jar:third_party/auto/auto-value-annotations-1.8.2.jar:third_party/auto/auto-common-1.1.2.jar:third_party/truth_proto/truth-proto-extension-1.0.1.jar:third_party/truth_proto/truth-liteproto-extension-1.0.1.jar:third_party/pcollections/pcollections-2.1.2.jar:third_party/jetifier/jetifier-core-1.0.0-beta04.jar:third_party/compile_testing/compile-testing-0.18.jar:third_party/auth/google-auth-library-oauth2-http-1.6.0.jar:third_party/auth/google-auth-library-credentials-1.6.0.jar:third_party/android_common/com.android_annotations_25.0.0.jar:third_party/android_common/com.android.tools.build_builder-test-api_2.0.0.jar:third_party/android_common/com.android.tools_sdk-common_25.0.0-patched-stripped.jar:third_party/android_common/com.android.tools.lint_lint-checks_25.0.0.jar:third_party/android_common/com.android.tools_dvlib_25.0.0.jar:third_party/android_common/com.android.tools.build_builder-model_2.0.0.jar:third_party/android_common/com.android.tools_sdklib_25.0.0.jar:third_party/android_common/com.android.tools.build_manifest-merger_25.0.0-patched.jar:third_party/android_common/com.android.tools_common_25.0.0.jar:third_party/android_common/com.android.tools.layoutlib_layoutlib_26.1.2-stripped.jar:third_party/android_common/com.android.tools_sdk-common_25.0.0-patched.jar:third_party/android_common/com.android.tools.layoutlib_layoutlib_26.1.2.jar:third_party/android_common/com.android.tools.build_builder_2.0.0.jar:third_party/android_common/com.android.tools.external.lombok_lombok-ast_0.2.3.jar:third_party/android_common/com.android.tools.lint_lint-api_25.0.0.jar:third_party/android_common/com.android.tools_repository_25.0.0.jar:third_party/opencensus/opencensus-api-0.24.0.jar:third_party/opencensus/opencensus-contrib-grpc-metrics-0.24.0.jar:third_party/gson/gson-2.8.0.jar:third_party/apache_commons_codec/commons-codec-1.10.jar:third_party/bytebuddy/byte-buddy-agent-1.9.7.jar:third_party/bytebuddy/byte-buddy-1.9.7.jar::third_party/grpc/grpc-core-1.41.0.jar:third_party/grpc/grpc-stub-1.41.0.jar:third_party/grpc/grpc-context-1.41.0.jar:third_party/grpc/grpc-protobuf-lite-1.41.0.jar:third_party/grpc/grpc-protobuf-1.41.0.jar:third_party/grpc/grpc-api-1.41.0.jar:third_party/grpc/grpc-netty-1.41.0.jar:third_party/grpc/grpc-auth-1.41.0.jar::third_party/guava/guava-30.1.1-jre.jar:third_party/guava/guava-testlib-30.1.1-jre.jar::/var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61 -sourcepath src/java_tools/singlejar/java/com/google/devtools/build/zip:src/main/java:tools/java/runfiles:/var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/src -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes -source 1.8 -target 1.8 -encoding UTF-8 @/var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_owYrdOr7/param
warning: [options] bootstrap class path not set in conjunction with -source 8
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:17: warning: Signal is internal proprietary API and may be removed in a future release
import sun.misc.Signal;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:18: warning: SignalHandler is internal proprietary API and may be removed in a future release
import sun.misc.SignalHandler;
               ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/DynamicCodec.java:30: warning: ReflectionFactory is internal proprietary API and may be removed in a future release
import sun.reflect.ReflectionFactory;
                  ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/EnumMapCodec.java:23: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:18: warning: [removal] AccessController in java.security has been deprecated and marked for removal
import java.security.AccessController;
                    ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:17: warning: Signal is internal proprietary API and may be removed in a future release
import sun.misc.Signal;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:18: warning: SignalHandler is internal proprietary API and may be removed in a future release
import sun.misc.SignalHandler;
               ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/DynamicCodec.java:30: warning: ReflectionFactory is internal proprietary API and may be removed in a future release
import sun.reflect.ReflectionFactory;
                  ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/EnumMapCodec.java:23: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:18: warning: [removal] AccessController in java.security has been deprecated and marked for removal
import java.security.AccessController;
                    ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:17: warning: Signal is internal proprietary API and may be removed in a future release
import sun.misc.Signal;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:18: warning: SignalHandler is internal proprietary API and may be removed in a future release
import sun.misc.SignalHandler;
               ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/DynamicCodec.java:30: warning: ReflectionFactory is internal proprietary API and may be removed in a future release
import sun.reflect.ReflectionFactory;
                  ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/EnumMapCodec.java:23: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:18: warning: [removal] AccessController in java.security has been deprecated and marked for removal
import java.security.AccessController;
                    ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:17: warning: Signal is internal proprietary API and may be removed in a future release
import sun.misc.Signal;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:18: warning: SignalHandler is internal proprietary API and may be removed in a future release
import sun.misc.SignalHandler;
               ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/DynamicCodec.java:30: warning: ReflectionFactory is internal proprietary API and may be removed in a future release
import sun.reflect.ReflectionFactory;
                  ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/EnumMapCodec.java:23: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:18: warning: [removal] AccessController in java.security has been deprecated and marked for removal
import java.security.AccessController;
                    ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:21: warning: Unsafe is internal proprietary API and may be removed in a future release
import sun.misc.Unsafe;
               ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:29: warning: Signal is internal proprietary API and may be removed in a future release
  private static final Signal SIGINT = new Signal("INT");
                       ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:29: warning: Signal is internal proprietary API and may be removed in a future release
  private static final Signal SIGINT = new Signal("INT");
                       ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:29: warning: Signal is internal proprietary API and may be removed in a future release
  private static final Signal SIGINT = new Signal("INT");
                       ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:29: warning: Signal is internal proprietary API and may be removed in a future release
  private static final Signal SIGINT = new Signal("INT");
                       ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:29: warning: Signal is internal proprietary API and may be removed in a future release
  private static final Signal SIGINT = new Signal("INT");
                                           ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:31: warning: SignalHandler is internal proprietary API and may be removed in a future release
  private SignalHandler oldHandler;
          ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:31: warning: SignalHandler is internal proprietary API and may be removed in a future release
  private SignalHandler oldHandler;
          ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:31: warning: SignalHandler is internal proprietary API and may be removed in a future release
  private SignalHandler oldHandler;
          ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:31: warning: SignalHandler is internal proprietary API and may be removed in a future release
  private SignalHandler oldHandler;
          ^
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:39: warning: Signal is internal proprietary API and may be removed in a future release
    this.oldHandler = Signal.handle(SIGINT, signal -&gt; run());
                      ^
warning: Signal is internal proprietary API and may be removed in a future release
src/main/java/com/google/devtools/build/lib/server/signal/InterruptSignalHandler.java:45: warning: Signal is internal proprietary API and may be removed in a future release
    Signal.handle(SIGINT, oldHandler);
    ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/DynamicCodec.java:276: warning: ReflectionFactory is internal proprietary API and may be removed in a future release
        ReflectionFactory.getReflectionFactory()
        ^
src/main/java/com/google/devtools/build/lib/skyframe/serialization/EnumMapCodec.java:59: warning: Unsafe is internal proprietary API and may be removed in a future release
      Unsafe unsafe = UnsafeProvider.getInstance();
      ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:38: warning: Unsafe is internal proprietary API and may be removed in a future release
  private final Unsafe unsafe;
                ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:38: warning: Unsafe is internal proprietary API and may be removed in a future release
  private final Unsafe unsafe;
                ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:38: warning: Unsafe is internal proprietary API and may be removed in a future release
  private final Unsafe unsafe;
                ^
src/main/java/com/google/devtools/build/lib/unsafe/StringUnsafe.java:38: warning: Unsafe is internal proprietary API and may be removed in a future release
  private final Unsafe unsafe;
                ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:31: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static final Unsafe UNSAFE = getUnsafe();
                       ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:31: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static final Unsafe UNSAFE = getUnsafe();
                       ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:31: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static final Unsafe UNSAFE = getUnsafe();
                       ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:31: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static final Unsafe UNSAFE = getUnsafe();
                       ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:33: warning: Unsafe is internal proprietary API and may be removed in a future release
  public static Unsafe getInstance() {
                ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:33: warning: Unsafe is internal proprietary API and may be removed in a future release
  public static Unsafe getInstance() {
                ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:33: warning: Unsafe is internal proprietary API and may be removed in a future release
  public static Unsafe getInstance() {
                ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:33: warning: Unsafe is internal proprietary API and may be removed in a future release
  public static Unsafe getInstance() {
                ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:43: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static Unsafe getUnsafe() {
                 ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:43: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static Unsafe getUnsafe() {
                 ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:43: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static Unsafe getUnsafe() {
                 ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:43: warning: Unsafe is internal proprietary API and may be removed in a future release
  private static Unsafe getUnsafe() {
                 ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:48: warning: Unsafe is internal proprietary API and may be removed in a future release
          new PrivilegedExceptionAction&lt;Unsafe&gt;() {
                                        ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:50: warning: Unsafe is internal proprietary API and may be removed in a future release
            public Unsafe run() throws Exception {
                   ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:51: warning: Unsafe is internal proprietary API and may be removed in a future release
              Class&lt;Unsafe&gt; unsafeClass = Unsafe.class;
                    ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:51: warning: Unsafe is internal proprietary API and may be removed in a future release
              Class&lt;Unsafe&gt; unsafeClass = Unsafe.class;
                                          ^
src/main/java/com/google/devtools/build/lib/unsafe/UnsafeProvider.java:47: warning: [removal] AccessController in java.security has been deprecated and marked for removal
      return AccessController.doPrivileged(
             ^
warning: Implicitly compiled files were not subject to annotation processing.
  Use -proc:none to disable annotation processing or -implicit to specify a policy for implicit compilation.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
65 warnings
.unzip -qn derived/jars/zstd-jni/libzstd-jni.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn derived/jars/com_google_protobuf/java/core/liblite.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn derived/jars/com_google_protobuf/java/core/libcore.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn derived/jars/com_google_protobuf/java/util/libutil.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_annotations/checker-qual-3.2.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_annotations/checker-qual-3.2.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/javax_annotations/javax.annotation-api-1.3.2-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/javax_annotations/javax.annotation-api-1.3.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/truth/truth-1.0.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libserialization-constant-annotation.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libserialization-constant-processor.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libserialization.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libautocodec-processor.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libunsafe-provider.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libautocodec-annotation.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libserialization-processor-util.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bazel_bootstrap/libregistered-singleton.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/turbine/turbine_direct.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jcip_annotations/jcip-annotations-1.0-1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_pool2/commons-pool2-2.8.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_compress/apache-commons-compress-1.19.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/animal_sniffer/animal-sniffer-annotations-1.14.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/flogger/flogger-system-backend-0.5.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/flogger/flogger-0.5.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/flogger/google-extensions-0.5.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jetbrains_annotations/annotations-13.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/truth8/truth-java8-extension-1.0.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/objenesis/objenesis-1_3.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/api_client/google-api-client-1.22.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/api_client/google-api-client-gson-1.22.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/api_client/google-http-client-1.22.0-SNAPSHOT.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/api_client/google-http-client-gson-1.22.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/tomcat_annotations_api/tomcat-annotations-api-8.0.5.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jimfs/jimfs-1.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/hamcrest/hamcrest-core-1.3.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/diffutils/diffutils-1.3.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/caffeine/caffeine-2.9.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_javacutil/javacutil-3.2.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_javacutil/javacutil-3.2.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jsr330_inject/javax.inject.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jdk/langtools/jdk_compiler-src.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jdk/langtools/jdk_compiler.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jdk/langtools/java_compiler.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jdk/langtools/java_compiler-src.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jdk/langtools/javac-9+181-r4173-1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/proguard/proguard6.2.2/lib/proguard.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/proguard/proguard6.2.2/lib/proguardgui.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/proguard/proguard6.2.2/lib/retrace.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/javapoet/javapoet-1.8.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/android_databinding/v3_4_0/base.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/android_databinding/v3_4_0/exec.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/android_databinding/v3_4_0/x_common.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/android_databinding/v3_4_0/compiler.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/android_databinding/v3_4_0/compiler_common.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.ant-0.8.7-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.agent-0.8.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.ant-0.8.7-nodeps.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.agent-0.8.7-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.core-0.8.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/jacocoagent-0.8.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.report-0.8.7-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.ant-0.8.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.report-0.8.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jacoco/org.jacoco.core-0.8.7-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/j2objc-annotations/j2objc-annotations-1.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java/jcommander/jcommander-1.48.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/juniversalchardet/juniversalchardet-1.0.3.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/guava/failureaccess-1.0.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_dataflow/dataflow-errorprone-3.15.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_dataflow/dataflow-shaded-3.11.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_dataflow/dataflow-errorprone-3.15.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/checker_framework_dataflow/dataflow-shaded-3.11.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jsr305/jsr-305.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-util-9.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-util-9.2-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-commons-9.2-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-tree-9.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-analysis-9.2-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-commons-9.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-analysis-9.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-9.2-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-tree-9.2-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/asm/asm-9.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_velocity/velocity-1.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/mockito/mockito-core-3.12.4.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/allocation_instrumenter/java-allocation-instrumenter-3.3.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/antlr/antlr4-4.5.3.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/perfmark/perfmark-api-0.19.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/error_prone/error_prone_core-2.9.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/error_prone/threeten-extra-1.5.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/error_prone/error_prone_type_annotations-2.9.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/error_prone/error_prone_annotation-2.9.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/error_prone/error_prone_annotations-2.9.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/error_prone/error_prone_check_api-2.9.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/libwrongcentraldir.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/jar-without-manifest.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/sealed/sealed.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/nestmates/nestmates.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/records/records.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/jar-with-manifest.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/ijar/test/jar-with-manifest-and-target-label.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/javax_activation/javax.activation-api-1.2.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/javax_activation/javax.activation-api-1.2.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_collections/commons-collections-3.2.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/xz/xz-1.9.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_lang/commons-lang-2.6.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/junit/junit-4.13.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/java-diff-utils/java-diff-utils-4.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/rxjava3/rxjava-3.1.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_logging/commons-logging-1.1.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/2.3.0/jaxb-core-2.3.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/2.3.0/jaxb-core-2.3.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/2.3.0/jaxb-api-2.3.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/2.3.0/jaxb-impl-2.3.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/2.3.0/jaxb-api-2.3.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/2.3.0/jaxb-impl-2.3.0-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/jaxb-api-2.3.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/jaxb-api-2.3.1-patched.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jaxb/jaxb-api-2.3.1-sources.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/hungarian_algorithm/software-and-algorithms-1.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/hungarian_algorithm/software-and-algorithms-1.0-src.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/kotlin_stdlib/kotlin-stdlib-jdk8-1.3.21.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/kotlin_stdlib/kotlin-stdlib-1.3.21.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/kotlin_stdlib/kotlin-stdlib-jdk7-1.3.21.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/kotlin_stdlib/kotlin-stdlib-common-1.3.21.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_io/commons-io-2.4.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/reactive_streams/reactive-streams-1.0.3.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-unix-common-4.1.69.Final-linux-aarch_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-handler-proxy-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-codec-http2-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-buffer-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-handler-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-unix-common-4.1.69.Final-osx-x86_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-epoll-4.1.69.Final-linux-x86_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-unix-common-4.1.69.Final-osx-aarch_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-unix-common-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-sctp-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-kqueue-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-unix-common-4.1.69.Final-linux-x86_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-common-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-resolver-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-epoll-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-resolver-dns-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-codec-http-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-epoll-4.1.69.Final-linux-aarch_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-kqueue-4.1.69.Final-osx-aarch_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-transport-native-kqueue-4.1.69.Final-osx-x86_64.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/netty/netty-codec-4.1.69.Final.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auto/auto-service-annotations-1.0-rc7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auto/auto-value-1.8.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auto/auto-service-1.0-rc7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auto/auto-value-annotations-1.8.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auto/auto-common-1.1.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/truth_proto/truth-proto-extension-1.0.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/truth_proto/truth-liteproto-extension-1.0.1.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/pcollections/pcollections-2.1.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/jetifier/jetifier-core-1.0.0-beta04.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/compile_testing/compile-testing-0.18.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auth/google-auth-library-oauth2-http-1.6.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/auth/google-auth-library-credentials-1.6.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android_annotations_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.build_builder-test-api_2.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools_sdk-common_25.0.0-patched-stripped.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.lint_lint-checks_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools_dvlib_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.build_builder-model_2.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools_sdklib_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.build_manifest-merger_25.0.0-patched.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools_common_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.layoutlib_layoutlib_26.1.2-stripped.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools_sdk-common_25.0.0-patched.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.layoutlib_layoutlib_26.1.2.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.build_builder_2.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.external.lombok_lombok-ast_0.2.3.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools.lint_lint-api_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/android_common/com.android.tools_repository_25.0.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/opencensus/opencensus-api-0.24.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/opencensus/opencensus-contrib-grpc-metrics-0.24.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/gson/gson-2.8.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/apache_commons_codec/commons-codec-1.10.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bytebuddy/byte-buddy-agent-1.9.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/bytebuddy/byte-buddy-1.9.7.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-core-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-stub-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-context-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-protobuf-lite-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-protobuf-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-api-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-netty-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/grpc/grpc-auth-1.41.0.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/guava/guava-30.1.1-jre.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
unzip -qn third_party/guava/guava-testlib-30.1.1-jre.jar -d /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes
./opt/openjdk-bin-17.0.3_p7/bin/jar cmf /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/MANIFEST.MF /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/libblaze.jar -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes A.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes AUTHORS -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes AnnotatedClass.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes Annotations$ParametersOnlyAnnotation.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes Annotations$RuntimeInvisible.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes Annotations$RuntimeVisible.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes Annotations.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes B.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes LICENSE -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes LICENSE-junit.txt -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes LICENSE.txt -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes MANIFEST.MF -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes NOTICE -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes NOTICE.txt -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes NestTest$P.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes NestTest.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes RecordTest.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes SealedTest$Bar.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes SealedTest$Foo.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes SealedTest.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes about.html -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes android -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes androidx -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes api-versions.xml -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes api_diff.proto -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes autovalue -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes blogspot -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes build -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes com -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes data_binding_version_info.properties -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes default.config -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes default.generated.config -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes deps.proto -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes difflib -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes io -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes jacocoagent.jar -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes javax -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes jdk -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes junit -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes kotlin -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes libzstd-jni.so -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes lombok -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes manifest.proto -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes module-info.class -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes module-info.java -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes net -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes org -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes pom.xml -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes proguard -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes sun -C /var/tmp/portage/dev-util/bazel-5.2.0/temp/bazel_bf8keP61/classes textfile.txt
..
Building Bazel with Bazel.
.OpenJDK 64-Bit Server VM warning: Options -Xverify:none and -noverify were deprecated in JDK 13 and will likely be removed in a future release.
INFO: Analyzed target //src:bazel_nojdk (367 packages loaded, 10170 targets configured).
INFO: Found 1 target...
[0 / 653] [Prepa] BazelWorkspaceStatusAction stable-status.txt
FATAL: bazel crashed due to an internal error. Printing stack trace:
java.lang.ExceptionInInitializerError
        at com.google.devtools.build.lib.actions.ParameterFile.writeContent(ParameterFile.java:118)
        at com.google.devtools.build.lib.actions.ParameterFile.writeParameterFile(ParameterFile.java:111)
        at com.google.devtools.build.lib.analysis.actions.ParameterFileWriteAction$ParamFileWriter.writeOutputFile(ParameterFileWriteAction.java:170)
        at com.google.devtools.build.lib.exec.FileWriteStrategy.beginWriteOutputToFile(FileWriteStrategy.java:58)
        at com.google.devtools.build.lib.analysis.actions.FileWriteActionContext.beginWriteOutputToFile(FileWriteActionContext.java:49)
        at com.google.devtools.build.lib.analysis.actions.AbstractFileWriteAction.beginExecution(AbstractFileWriteAction.java:66)
        at com.google.devtools.build.lib.actions.Action.execute(Action.java:133)
        at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor$5.execute(SkyframeActionExecutor.java:907)
        at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor$ActionRunner.continueAction(SkyframeActionExecutor.java:1076)
        at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor$ActionRunner.run(SkyframeActionExecutor.java:1031)
        at com.google.devtools.build.lib.skyframe.ActionExecutionState.runStateMachine(ActionExecutionState.java:152)
        at com.google.devtools.build.lib.skyframe.ActionExecutionState.getResultOrDependOnFuture(ActionExecutionState.java:91)
        at com.google.devtools.build.lib.skyframe.SkyframeActionExecutor.executeAction(SkyframeActionExecutor.java:492)
        at com.google.devtools.build.lib.skyframe.ActionExecutionFunction.checkCacheAndExecuteIfNeeded(ActionExecutionFunction.java:856)
        at com.google.devtools.build.lib.skyframe.ActionExecutionFunction.computeInternal(ActionExecutionFunction.java:349)
        at com.google.devtools.build.lib.skyframe.ActionExecutionFunction.compute(ActionExecutionFunction.java:169)
        at com.google.devtools.build.skyframe.AbstractParallelEvaluator$Evaluate.run(AbstractParallelEvaluator.java:590)
        at com.google.devtools.build.lib.concurrent.AbstractQueueVisitor$WrappedRunnable.run(AbstractQueueVisitor.java:382)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make java.lang.String(byte[],byte) accessible: module java.base does not "opens java.lang" to unnamed module @3e07d849
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Constructor.checkCanSetAccessible(Constructor.java:188)
        at java.base/java.lang.reflect.Constructor.setAccessible(Constructor.java:181)
        at com.google.devtools.build.lib.unsafe.StringUnsafe.&lt;init&gt;(StringUnsafe.java:75)
        at com.google.devtools.build.lib.unsafe.StringUnsafe.initInstance(StringUnsafe.java:56)
        at com.google.devtools.build.lib.unsafe.StringUnsafe.&lt;clinit&gt;(StringUnsafe.java:37)
        ... 21 more
```

```
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make java.lang.String(byte[],byte) accessible: module java.base does not "opens java.lang" to unnamed module @3e07d849
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:354)
        at java.base/java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:297)
        at java.base/java.lang.reflect.Constructor.checkCanSetAccessible(Constructor.java:188)
        at java.base/java.lang.reflect.Constructor.setAccessible(Constructor.java:181)
        at com.google.devtools.build.lib.unsafe.StringUnsafe.&lt;init&gt;(StringUnsafe.java:75)
```

This is https://openjdk.org/jeps/403

The workaround is to pass `--add-opens=java.base/java.nio=ALL-UNNAMED` as a server JVM flag. The bazel launcher already passes that flag, but only for embedded JDKs:

https://github.com/bazelbuild/bazel/blob/03ad0cd1203b7b54ec02ea3f5788d1d673a9d6d6/src/main/cpp/blaze.cc#L362-L367

--- a/src/main/cpp/blaze.cc
+++ b/src/main/cpp/blaze.cc
@@ -359,13 +359,10 @@ static vector<string> GetServerExeArgs(const blaze_util::Path &jvm_path,
   startup_options.AddJVMArgumentPrefix(jvm_path.GetParent().GetParent(),
                                        &result);
 
-  // TODO(b/109998449): only assume JDK >= 9 for embedded JDKs
-  if (!startup_options.GetEmbeddedJavabase().IsEmpty()) {
-    // quiet warnings from com.google.protobuf.UnsafeUtil,
-    // see: https://github.com/google/protobuf/issues/3781
-    result.push_back("--add-opens=java.base/java.nio=ALL-UNNAMED");
-    result.push_back("--add-opens=java.base/java.lang=ALL-UNNAMED");
-  }
+  // com.google.devtools.build.lib.unsafe.StringUnsafe uses reflection to access
+  // private fields in java.lang.String. The Bazel server requires Java 11, so
+  // this option is known to be supported.
+  result.push_back("--add-opens=java.base/java.lang=ALL-UNNAMED");
 
   result.push_back("-Xverify:none");
 

